FluentUI/src/main/java/com/microsoft/fluentui/persona/AvatarGroupView.kt

```diff
@@ -0,0 +1,162 @@
+package com.microsoft.fluentui.persona
+
+import android.content.Context
+import android.support.v4.content.ContextCompat
+import android.util.AttributeSet
+import android.view.View
+import android.view.ViewGroup
+import android.widget.FrameLayout
+import com.microsoft.fluentui.R
+
+
+enum class AvatarGroupStyle {
+    STACK, PILE
+}
+
+open class AvatarGroupView : FrameLayout {
+    companion object {
+        internal val DEFAULT_AVATAR_GROUP_STYLE = AvatarGroupStyle.STACK
+        internal const val DEFAULT_AVATAR_ALLOWED = 4
+    }
+
+    @JvmOverloads
+    constructor(context: Context, attrs: AttributeSet? = null, defStyleAttr: Int = 0) : super(context, attrs, defStyleAttr) {
+        val styledAttrs = context.obtainStyledAttributes(attrs, R.styleable.AvatarGroupView)
+        val avatarSizeOrdinal = styledAttrs.getInt(R.styleable.AvatarGroupView_avatarSize, AvatarView.DEFAULT_AVATAR_SIZE.ordinal)
+        val avatarGroupStyleOrdinal = styledAttrs.getInt(R.styleable.AvatarGroupView_avatarGroupStyle, DEFAULT_AVATAR_GROUP_STYLE.ordinal)
+
+        avatarSize = AvatarSize.values()[avatarSizeOrdinal]
+        avatarGroupStyle = AvatarGroupStyle.values()[avatarGroupStyleOrdinal]
+        maxDisplayedAvatars = styledAttrs.getInt(R.styleable.AvatarGroupView_maxDisplayedAvatars, DEFAULT_AVATAR_ALLOWED)
+        styledAttrs.recycle()
+    }
+
+    private var avatarList: List<IAvatar> = ArrayList()
+    var listener: Listener? = null
```

Comment:
private ?

Reply:
this is exposed, we do not provide a setter,  consumers can directly set value in this